testing/oak_echo_bin/README.md

```diff
@@ -1,4 +1,7 @@
-# Echo Baremetal
+# Oak Echo
 
 Echo baremetal binary that can be run under
 [crosvm](https://chromium.googlesource.com/chromiumos/platform/crosvm/).
+
+This binary uses IDL to communicate between the Untrusted Launcher and the guest
+VM.
```

Comment:
Could you add a link to the actual IDL definition for this binary?

Reply:
Done